lldb/include/lldb/Core/Debugger.h

```diff
@@ -83,7 +83,7 @@ class Debugger : public std::enable_shared_from_this<Debugger>,
     eBroadcastBitProgress = (1 << 0),
     eBroadcastBitWarning = (1 << 1),
     eBroadcastBitError = (1 << 2),
-    eBroadcastSymbolChange = (1 << 3),
+    eBroadcastBitSymbolChange = (1 << 3),
```

Comment:
This name led me to believe that a Symbol from a symbol table was changed, so it confused me just reading the code. Do we want this to be "eBroadcastBitSymbolFileAdded"?

Reply:
If we change the name of the broadcast bit, we should also rename the `SymbolChangeEventData`.

Reply:
Should it be `SymbolFileAdded` or `SymbolFileChanged`? You can (re)load a symbol file, which is something I've done with the JSON symbol file format.

Reply:
> Should it be `SymbolFileAdded` or `SymbolFileChanged`? You can (re)load a symbol file, which is something I've done with the JSON symbol file format.

I am fine with any name as long as it is clear that this is a symbol _file_ change. The previous name led me to think a new symbol showed up in some symbol table. Luckily this enum is private, so we can rename as needed. But we should get the name as we want it in the SBDebugger.h file since we can't rename that after releasing it in the public API.

Reply:
For clarity it can be renamed to `eBroadcastBitSymbolFilesChanged` in Debugger.h since we should remove this flag from the public view.

Reply:
Yes, that's what Greg's suggesting 👍
PS: Probably just a typo, but it should be `eBroadcastBitSymbolFileChanged` (instead of `Files`).

Reply:
Yes, I mistyped but fixed the typo in the fixup commit.